packages/client/src/Dashboard.tsx

```diff
@@ -452,15 +447,15 @@ function ConversationRow({
           secondary: classes.snippetText
         }}
       />
-      {newLabels &&
-        newLabels.map(label => (
-          <DisplayLabel key={labelId + count++} label={label} />
-        ))}
+      {labels &&
+        labels.map(label => <DisplayLabel key={label} label={label} />)}
     </ListItem>
   )
 }
 
 function DisplayLabel({ label }: { label: string }) {
+  const labelForDisplay = label.replace(/\\/, "")
```

Comment:
Yes, much better! There is one tweak to be made: this replacement should only be made if the backslash is the first character in the label string. You can restrict a regular expression pattern to the beginning of a string using the `^` anchor:

```ts
label.replace(/^\\/, "")
```